assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -16,7 +16,7 @@ import { siteTheme } from '../../globals'
 const accessedResourceColor = siteTheme.palette.secondary.main
 const notAccessedResourceColor = siteTheme.palette.negative.main
 const linkColor = siteTheme.palette.link.main
-const mainMargin = { top: 50, right: 10, bottom: 50, left: 200 }
+const foreignObjSide = 24
```

Comment:
How do you come up with this number? Is it based on anything else? What would happen if someone changed it? Could you add a comment here as it's not obvious to me what this does from this value, the name of the variable or the code.

Reply:
The number is arbitrary. It specifies the side length of the square foreign object element, which contains the resource icon and the spacing to its right. Originally you'll see on the old lines 376 and 377 that it was 32. I think I made it a little smaller since the spacing between the icon and resource name was fairly large. This was created and moved up since the value is also needed for determining the placement of the resource label. I will add a comment.